app/javascript/packs/application.js

```diff
@@ -2,25 +2,56 @@
 // present in this directory. You're encouraged to place your actual application logic in
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
-import 'bootstrap'
-import './src/application.scss'
+import "bootstrap";
+import "./src/application.scss";
 
-require("@rails/ujs").start()
-require("turbolinks").start()
-require("@rails/activestorage").start()
-require("channels")
+require("@rails/ujs").start();
+require("turbolinks").start();
+require("@rails/activestorage").start();
+require("channels");
+require("jquery");
 
 document.addEventListener("turbolinks:load", () => {
-  $('[data-toggle="tooltip"]').tooltip()
-  $('[data-toggle="popover"]').popover()
-  $('.toast').toast({ autohide: false })
-  $('#toast').toast('show')
-})
+  $('[data-toggle="tooltip"]').tooltip();
+  $('[data-toggle="popover"]').popover();
+  $(".toast").toast({ autohide: false });
+  $("#toast").toast("show");
+});
 
+$(document).ready(function() {
+  $(".showCommentbtn").click(function() {
+    console.log("CALLED showCommentbtn chk --0");
+    var btnId = $(this).attr("id");
+    $(".commentArea").each(function(index, value) {
+      var jsonInfoDataId = $(this).data("id");
+      if (btnId == jsonInfoDataId) {
```

Comment:
Prefer using `===` for comparing, its faster in execution
You can read more about it [here](https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript/)

Reply:
and the condition should:
```js
if (btnId && btnId == jsonInfoDataId) {
```

Always look for the cases where your code may throw error due to missing params. Try to be futuristic about it